tests/controls/gridtest.cpp

```diff
@@ -31,32 +31,165 @@
 namespace
 {
 
-// Derive a new class inheriting from wxGrid just to get access to its
+wxString CellCoordsToString(int row, int col)
+{
+    return wxString::Format("(%d, %c)", 1 + row, 'A' + col);
```

Comment:
Minor, but in the grid sample we use `R%dC%d` format which I think is also quite clear, so I'd use it here too for consistency.

If not, i.e. if we want to use the most usual format, then I'd use just "A1" format, i.e. remove the parentheses and the comma and reverse the components.

Reply:
I plan on using the latter because a grid's default row and column labels are shown when rendering (I find those less confusing than using only digits for the coords), so currently it's not 0-based or even using numbers for columns. The `(r, c)` format came from related trac tickets I think and documentation.